src/waveform/renderers/waveformwidgetrenderer.cpp

```diff
@@ -47,6 +47,10 @@ WaveformWidgetRenderer::WaveformWidgetRenderer(const char* group)
     m_timer = new QTime();
     currentFrame = 0;
     m_lastFrameTime = 0;
+    for (int i = 0; i < 100; ++i) {
+        m_lastFramesTime[i] = 0;
+        m_lastFramesTime[i] = 0;
```

Comment:
The same error is in the loop below. I think it is a copy and past error in the original code.